src/components/parameters/Parameters.svelte.test.ts

```diff
@@ -1,8 +1,16 @@
 import { cleanup, render } from '@testing-library/svelte';
-import { afterEach, describe, expect, it } from 'vitest';
+import { afterEach, describe, expect, it, vi } from 'vitest';
 import type { FormParameter } from '../../types/parameter';
 import Parameters from './Parameters.svelte';
 
+vi.mock('$env/dynamic/public', () => {
```

Comment:
Is this mock needed? The `Parameter` components don't seem to be using env vars.